.github/actions/lint/action.yml

```
@@ -31,36 +31,58 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Setup NodeJs Extensions
+    - name: Set up Environment
+      id: set-up-environment
       shell: pwsh
       run: |
         '${{ github.action_path }}' >> $Env:GITHUB_PATH
 
-        $LombiqNodeJsExtensions = $(Resolve-Path '${{ github.action_path }}/../../../Lombiq.NodeJs.Extensions').Path
-        Write-Output "LOMBIQ_NODEJS_EXTENSIONS=$LombiqNodeJsExtensions"
-        "LOMBIQ_NODEJS_EXTENSIONS=$LombiqNodeJsExtensions" >> $Env:GITHUB_ENV
+        $nodeJsExtensionsPath = $(Resolve-Path '${{ github.action_path }}/../../../Lombiq.NodeJs.Extensions').Path
+        Write-Output "LOMBIQ_NODEJS_EXTENSIONS=$nodeJsExtensionsPath"
+        "LOMBIQ_NODEJS_EXTENSIONS=$nodeJsExtensionsPath" >> $Env:GITHUB_ENV
         "LOMBIQ_NODEJS_EXTENSIONS_GITHUB_ACTIONS=true" >> $Env:GITHUB_ENV
         "LOMBIQ_NODEJS_EXTENSIONS_GITHUB_ACTIONS_ROOT=$PWD" >> $Env:GITHUB_ENV
         "LOMBIQ_NODEJS_EXTENSIONS_WARN_AS_ERROR=true" >> $Env:GITHUB_ENV
 
-        cd $LombiqNodeJsExtensions
+        # Read the packageManager from the package.json to be able to provide it to the pnpm setup action. While it can
+        # read it from the file, it can't actually seem to be able to access the file when the lint workflow is run from
+        # a parent workflow.
```

```suggestion
        # read it from the file, it can't access the file when the lint workflow is run from a parent workflow.
```
Do those words contribute anything to this note? Seems like filler to me.

---

Not much, yeah.